dashboard/app/models/script.rb

```diff
@@ -151,9 +151,9 @@ def starting_level
   @@script_cache = nil
   SCRIPT_CACHE_KEY = 'script-cache'
 
-  # Caching is disabled when editing scripts and levels.
+  # Caching is disabled when editing scripts and levels or running unit tests on CI.
   def self.should_cache?
-    !Rails.application.config.levelbuilder_mode
+    !(Rails.application.config.levelbuilder_mode || ENV['CI'])
```

Comment:
This would be cleaner as
```
return false if Rails.application.config.levelbuilder_mode
return fase if ENV['CI']
true
```
as otherwise I need to parse the negation and disjunction, as well as determine the two conditions have nothing to do with each other.